R/groupwise_abs_diff.R

```diff
@@ -0,0 +1,24 @@
+#' Groupwise Absolute Differenence (Fairness Metrics)
+#' @name groupwise_abs_diff
+#'
+#' @param prediction The predictions of the learner.
+#' @param base_measure The base measures used to evaluate.
+#' @param data_task The data task for the fairness metric.
+#' @param ... Further arguments, currently ignored.
+#'
+#' @return Absolute difference of base measues between binary protected groups.
```

Comment:
We always document the data type of params and outputs, see above.

Reply:
Fixed